app/mailers/report_mailer.rb

```diff
@@ -5,10 +5,27 @@ def budget_status
     to_email = ENV.fetch('BUDGET_REPORT_RECIPIENT')
     date = Time.now.utc.strftime("%a %m/%d/%y (%Z)")
 
+    build_attachments
+
     mail(
       to: to_email,
       subject: "C2: Daily Budget report for #{date}",
       from: self.sender_email
     )
   end
+
+  # rubocop:disable all
```

Comment:
That's cheating! In all seriousness, we have Rubocop enabled expressly for the purpose of keeping us honest. Please don't disable!